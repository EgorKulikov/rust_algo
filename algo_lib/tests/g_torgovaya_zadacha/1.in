3 4 5
10 30 15
12 31 14 18
0 1 2 3 4
