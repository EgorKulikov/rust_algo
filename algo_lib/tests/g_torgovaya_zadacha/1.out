55
56
60
64
64
